tests/acceptance/00_basics/ifelapsed_and_expireafter/locks.cf.sub

```diff
@@ -0,0 +1,52 @@
+# Test that locking works
+# Called twice from main.cf
```

Comment:
i wuld jiust clarify this to say that cf-agent is run with this policy as the entry twice. so that there isn't confusion that a bundle in here is called twice within one agent run.

Reply:
Right.